drivers/esolar/driver.compose.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": {
+        "en": "eSolar WiFi Module",
```

Comment:
How does this fit in the SAJ ecosystem? Will every SAJ inverter have either LAN support, or WiFi support? Are eSolar modules only used with SAJ inverters?

If the answer to both questions is yes, then I would call this `SAJ (eSolar WiFi)` and the existing one `SAJ (LAN)`.